pydecred/rpc.py

```diff
@@ -585,6 +734,236 @@ def get(k, obj):
+            height=obj["height"],
+            blockHash=obj["hash"],
+            branchLen=obj["branchlen"],
+            status=obj["status"],
+        )
+
+
+class GetBlockSubsidyResult:
+    """getblocksubsidyresult"""
+
+    def __init__(
+        self, developer, subsidyPoS, subsidyPoW, total,
+    ):
+        """
+        Args:
+            developer (int): The developer subsidy.
+            subsidyPoS (int): The Proof-of-Stake subsidy.
+            subsidyPoW (int): The Proof-of-Work subsidy.
+            total (int): The total subsidy.
+        """
+        self.developer = developer
+        self.subsidyPoS = subsidyPoS
+        self.subsidyPoW = subsidyPoW
+        self.total = total
+
+    @staticmethod
+    def parse(obj):
+        """
+        Parse the GetBlockSubsidyResult from the decoded RPC response.
+
+        Args:
+            obj (object): The decoded dcrd RPC response.
+
+        Returns:
+            GetBlockSubsidyResult: The GetBlockSubsidyResult.
+        """
+        return GetBlockSubsidyResult(
+            developer=obj["developer"],
+            subsidyPoS=obj["pos"],
+            subsidyPoW=obj["pow"],
+            total=obj["total"],
+        )
+
+
+class GetCFilterV2Result:
+    def __init__(
+        self, blockHash, data, proofIndex, proofHashes,
+    ):
+        """
+        Args:
+            blockHash (str): The block hash for which the filter includes data.
+            data (str): Hex-encoded bytes of the serialized filter.
+            proofIndex (int): The index of the leaf that represents the filter
+                hash in the header commitment.
+            proofHashes (list(str)): The hashes needed to prove the filter is
+                committed to by the header commitment.
+        """
+        self.blockHash = blockHash
+        self.data = data
+        self.proofIndex = proofIndex
+        self.proofHashes = proofHashes
+
+    @staticmethod
+    def parse(obj):
+        """
+        Parse the GetCFilterV2Result from the decoded RPC response.
+
+        Args:
+            obj (object): The decoded dcrd RPC response.
+
+        Returns:
+            GetCFilterV2Result: The GetCFilterV2Result.
+        """
+        return GetCFilterV2Result(
+            blockHash=obj["blockhash"],
+            data=obj["data"],
+            proofIndex=obj["proofindex"],
+            proofHashes=obj["proofhashes"],
+        )
+
+
+class GetBlockHeaderVerboseResult:
+    """getblockheaderverboseresult"""
+
+    def __init__(
+        self,
+        blockHash,
+        confirmations,
+        version,
+        merkleRoot,
+        stakeRoot,
+        voteBits,
+        finalState,
+        voters,
+        freshStake,
+        revocations,
+        poolSize,
+        bits,
+        sBits,
+        height,
+        size,
+        time,
+        nonce,
+        extraData,
+        stakeVersion,
+        difficulty,
+        chainWork,
+        previousHash=None,
+        nextHash=None,
```

Comment:
It is really tempting just to inherit from `msgblock.BlockHeader` for this type. `BlockHeader` doesn't actually do anything in its constructor, but it could, and all its fields are also here.

Reply:
Now you can get the msgblock.BlockHeader from verbose=False. This does have five extra fields, and I guess the lengths of all the ints would need to be set if we inherit?